src/psql_migration.erl

```diff
@@ -176,20 +176,33 @@ connection_opts(_Args, {url, DatabaseUrl}) ->
     case http_uri:parse(DatabaseUrl, ParseOpts) of
         {error, Error} ->
             {error, Error};
-        {ok, {_, UserPass, Host, Port, Database, _}} ->
+        {ok, {_, UserPass, Host, Port, Database, Query}} ->
             {User, Pass} = case string:split(UserPass, ":") of
                 [[]] -> {"postgres", ""};
                 [U] -> {U, ""};
                 [[], []] -> {"postgres", ""};
                 [U, P] ->  {U, P}
             end,
 
-            {ok, #{
+            ConnectionOpts = #{
                 port => Port,
                 username => User,
                 password => Pass,
                 host => Host,
-                database => string:slice(Database, 1)}}
+                database => string:slice(Database, 1)},
+
+            case Query of
+                [] -> {ok, ConnectionOpts};
+                "?" ++ QueryString ->
+                    case uri_string:dissect_query(QueryString) of
+                        [] -> {ok, ConnectionOpts};
+                        QueryList ->
+                            case proplists:get_value("ssl", QueryList) of
+                                "true" -> {ok, maps:put(ssl, true, ConnectionOpts)};
```

Comment:
why not just merge a map made from the proplist in with ConnectionOpts?

Reply:
Would be fine if I trusted developers like me to not put in other garbage in the url string. I’d rather just extract what’s expected.

Reply:
Except developers supply the url too? And may have more options than we support out of the box? What are we protecting against?

Reply:
Typos mostly. I tend towards defensiveness but it’s a weakly held belief. Will make the change.

Reply:
One nice upside of map merging is that we can support ssl=required.

Reply:
yeah, I'm with you on only opening the holes one at a time. but that nice upside may make it worth it?

Reply:
Other downside of a simple map merge is that we’d have to convert to atoms for ssl and maybe other types for other fields

Reply:
ah yeah, fair enough.. let's keep it like this for now

Reply:
My madness sometimes has a method. Even if I don’t know it?

Reply:
I'd have probably figured out a mad way to get a map done, but it's just not the simplest thing to do here :-)